javaagent-tooling/opentelemetry-ext-noop-api-for-instrumenting/build.gradle.kts

```diff
@@ -0,0 +1,25 @@
+plugins {
+  id("com.github.johnrengelman.shadow")
+
+  id("otel.java-conventions")
+  id("otel.publish-conventions")
+}
+
+group = "io.opentelemetry.javaagent"
+
+dependencies {
+  implementation("io.opentelemetry:opentelemetry-extension-noop-api")
+}
+
+tasks {
+  shadowJar {
+    dependencies {
+      include(dependency("io.opentelemetry:opentelemetry-extension-noop-api"))
```

Comment:
ShadowJar automatically uses runtimeClasspath, including `implementation` dependencies. Why do we need to list it here again?

Reply:
To exclude transitive dependencies of `opentelemetry-extension-noop-api`.